music_assistant/providers/opensubsonic/parsers.py

```diff
@@ -48,11 +45,11 @@ def parse_artist(
     )
 
     artist.metadata.images = UniqueList()
-    if sonic_artist.cover_id:
+    if sonic_artist.coverArt:
```

Comment:
is the non-pythonic camelcase name intended ?

Reply:
Yes, both of these py-opensonic objects changed to camelCase, this is an unfortunate side effect of the move to dataclasses. The Open Subsonic spec inherited the camel casing from the original spec and they chose to maintain backwards compatibility, even when the original spec was broken. See: https://opensubsonic.netlify.app/docs/responses/playlistwithsongs/ as an example. I could rename all these fields, but (happy to be wrong here) that would require me doing hand rolled parsing and the point of moving to dataclasses was to drop that.

Reply:
Well, then you can use aliases to always do the conversion for you. Look at the library @fmunkes just created for ABS for example

Reply:
I'll be back, good thing there are only 300 or so fields that need annotating.

Me editorializing: this snake vs camel case is the first time I have seen a 'pythonic' thing that is genuinely silly as opposed to rooted in a real concern. Don't get me wrong, I am here for a 0 consequence religious war (vi for life) but to make it a part of the 'one true way' for the language is a bit extra.

Reply:
it is just a style standard and we (like many other projects) follow the PEP styles - it is ok to follow an existing library which doesn't have all the styles but if you want to setup a library from scratch today, snake case it is.

Reply:
Okay, so the newly pushed commits should do it. The entire library is now PEP style compliant.